applications/StructuralMechanicsApplication/python_scripts/kratos_main_structural.py

```diff
@@ -57,9 +57,11 @@
 
 ## Print model_part and properties
 if ((parallel_type == "OpenMP") or (mpi.rank == 0)) and (echo_level > 1):
-    Logger.PrintInfo(main_model_part)
+    Logger.PrintInfo("ModelPart", main_model_part)
+    count = 0
     for properties in main_model_part.Properties:
-        Logger.PrintInfo(properties)
+        count += 1
+        Logger.PrintInfo("Property " + str(count), properties)
```

Comment:
You may use the Property id here which is more understable from user point of view

Reply:
Makes sense